content/sensu-go/6.8/operations/deploy-sensu/secure-postgres.md

```diff
@@ -0,0 +1,422 @@
+---
+title: "Secure PostgreSQL"
+linkTitle: "Secure PostgreSQL"
+guide_title: "Secure PostgreSQL"
+type: "guide"
+description: "Learn how to secure communication between Sensu and PostgreSQL with certificate authentication."
+weight: 65
+version: "6.8"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-6.8:
+    parent: deploy-sensu
+---
+
+This guide describes how to secure communication between Sensu and the PostgreSQL event store using certificate authentication.
+When deploying Sensu for use outside of a local development environment, you should secure it using transport layer security (TLS).
+
+To learn how to secure communications between Sensu and its agents, read [Generate certificates for your Sensu installation][3] and [Secure Sensu][1].
+
+{{% notice note %}}
+**NOTE**: This guide describes one option for securing communication between Sensu and PostgreSQL and is intended as a starting point.
+Your organization's needs may require a different approach.
+{{% /notice  %}}
+
+## Prerequisites
+
+To use this guide, you must have:
+
+- A running Sensu deployment.
+- A running PostgreSQL instance.
+
+## Install cfssl
+
+The [CloudFlare cfssl][2] toolkit is released as a collection of command-line tools.
+
+If you followed [Generate certificates for your Sensu installation][3], you already downloaded and installed the Cloudflare cfssl toolkit.
+If not, run the following commands:
+
+{{< language-toggle >}}
+
+{{< code "RHEL/Rocky/Alma" >}}
```

Comment:
I'm uneasy with using `RHEL/Rocky/Alma` instead of `RHEL/CentOS` in this doc because it's inconsistent with the rest of the docs and https://sensu.io/downloads. I think we should discuss whether to make this update globally before making it here.

Reply:
Understandable--I'm personally in favor of making that update docs-wide given that CentOS is no longer being actively maintained and has forked to Rocky/Alma. I'm happy to change it back and we can revisit.